#426 のマージ後、このブランチを `daisskey` へ rebase して押し直しました（`5177c03e3f`）。
submodule のエラーは解消しています。

⚠ **ただし CI はまだ赤で、原因は 1 枚めくれた別の欠落でした。**やはりこの変更とは無関係です:

```
cp: cannot stat '.github/misskey/test.yml': No such file or directory
##[error]Process completed with exit code 1.
```

`.github/misskey/test.yml` がツリーに無いのに `frontend-diagnostics.inspect.yml:100,115` と
`backend-diagnostics.inspect.yml:70,84` が `cp` しています。⚠⚠ **診断系 4 本すべてが
この理由で動いておらず**（`inspect` は毎回 failure、`comment` は毎回 skipped）、
このフォークの PR で実際に走る CI は `packages-private.yml` 1 本だけ、という状態です。

#428 に起票しました。⚠ **この PR は #428 が片付くまで緑になりません。**変更自体は
`WidgetTagset.vue` の 5 行で、新しい識別子も型も増えていないので、レビューは進められます。

--- a/packages/frontend/src/widgets/WidgetTagset.vue
+++ b/packages/frontend/src/widgets/WidgetTagset.vue
@@ -103,10 +103,11 @@ const buildLabel = (p: Program, now: Date): string => {
 		label.push(`${dic.episodePrefix}${p.episode}${p.episode_suffix || dic.episodeSuffix}`);
 	}
 	if (p.subtitle) label.push(`「${p.subtitle}」`);
-	if (p.livecure) {
-		if (p.air) label.push(dic.air);
-		label.push(dic.livecure);
-	}
+	// air は livecure とは独立した軸（モロヘイヤ docs/api.md）。従属させると、下の
+	// user_tags 側（無条件）と食い違い、確認ダイアログで見た文字列と実際に送るタグが
+	// ズレる。capsicum / Mastodon フォークも無条件 (#424)
+	if (p.air) label.push(dic.air);
+	if (p.livecure) label.push(dic.livecure);
 	if (p.minutes) label.push(`${p.minutes}分`);
 	(p.extra_tags ?? []).forEach(tag => label.push(tag));
 	return label.join(' ');

